hi-ml-histopathology/src/histopathology/utils/report_utils.py

```diff
@@ -15,8 +15,51 @@
                                 get_tags_from_hyperdrive_run)
 
 
+def run_has_val_and_test_outputs(run: Run) -> bool:
+    """Checks whether the given run has both validation and test outputs files.
+
+    :param parent_run: The run whose outputs to check.
+    :raises ValueError: If the run does not have the expected output file(s).
+    :return: `True` if the run has validation and test outputs, `False` if it is a legacy run with
+        only test outputs.
+    """
+    outputs_filename = "test_output.csv"
+    val_outputs_filename = "val/" + outputs_filename
+    test_outputs_filename = "test/" + outputs_filename
+
+    outputs_folder = "outputs/"
+    all_available_files: List[str] = run.get_file_names()
+    output_files = [filename[len(outputs_folder):] for filename in all_available_files
```

Comment:
Is it simpler to set `val_outputs_filename = "outputs/val/" + outputs_filename` and then let this condition be `if val_ouputs_filename in all_available_files`? (and similarly for test_outputs_filename)